### ISSUE TITLE
Is there any direct support for teh SAMD51 M4 Chip

### ISSUE THREAD

**Issue by trlafleur** [ttlappalainen/NMEA2000#196]
Adafruit just release a Feather M4 CAN board with transceiver, curious if a SAMD51 driver is available for this processor.

thanks

**Comment by ttlappalainen** [comment]
No. Note that tranceiver is either not isolated and so not suitable in many solutions.

**Comment by skarlsson** [comment]
very late in thread - but a couple of months ago I throw together something for that chip - it seems to work well  (I have a built a few devices) https://github.com/skarlsson/NMEA2000_FeatherM4CAN
